examples/vpc_service_control/onprem_project/variables.tf

```
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+variable "project_id" {}
+
+variable "service_account_file" {}
+
+variable "organization_id" {}
+
+variable "billing_account_id" {}
+
+variable "cloud_router_region" {}
```

Since this variable is used pretty much everywhere not just for the router, it should simply be named region.
Also, a default for every variable that can have a default to reduce the setup time for users.
```suggestion
variable "region" {
description = "Region to deploy all resources in"
default = "us-central1"
}
```

---

Done.

---

The variable is still missing a default and a description.